controllers/repositories_controller.go

```diff
@@ -0,0 +1,50 @@
+package controllers
+
+import (
+	"../models/hook"
+	"../models/repository"
+	"../modules/logging"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/google/go-github/github"
+	"github.com/zenazn/goji/web"
+)
+
+type Repositories struct {
+}
+
+// webhook event from github
```

Comment:
__[golint]__
* [golint] comment on exported method Repositories.Hook should be of the form "Hook ..."